docs/04-operation-guides/troubleshooting/evnt-01-eventing-troubleshooting.md

```diff
@@ -1,44 +1,125 @@
 ---
-title: Did not receive an event - basic diagnostics
+title: Kyma Eventing - Basic Diagnostics
 ---
 
 ## Symptom
 
-You publish an event but the event is not received by the subscriber.
+- You publish an event but the event is not received by the subscriber.
+
+- Subscription is not in ready state.
+
+- You are unable to publish events.
 
 ## Remedy
 
 Follow these steps to detect the source of the problem:
 
-1. Check the Eventing backend CRD, if the field `EVENTINGREADY` is true:
+### Step 1: Check the status of the Eventing backend CR
+
+1. Check the Eventing backend CR. Is the field **EVENTINGREADY** `true`?
    
-   ```bash
-   kubectl -n kyma-system get eventingBackend
-   ```
-   If `EVENTINGREADY` is false, check the exact reason of the error in the status of the CRD by running the command:
+    ```bash
+    kubectl -n kyma-system get eventingbackends.eventing.kyma-project.io
+    ```
+
+2. If `EVENTINGREADY` is false, check the exact reason of the error in the status of the CR by running the command:
+
+    ```bash
+    kubectl -n kyma-system get eventingbackends.eventing.kyma-project.io eventing-backend -o yaml
+    ```
+
+3. If `EVENTINGREADY` is true, Eventing backend is not an issue. Follow the next steps to find the source of the problem.
+
+### Step 2: Check the status of the Subscription
+
+1. Check whether the Subscription is ready. Run the command:
+
+    ```bash
+    kubectl -n {NAMESPACE} get subscriptions.eventing.kyma-project.io {NAME}
+    ```
 
-   ```bash
-   kubectl -n kyma-system get eventingBackend eventing-backend -o yaml
-   ```
+2. If the Subscription is not ready, check the exact reason of the error in the status of the Subscription by running the command:
```

Comment:
hmm..step 2.3 would not be relevant, but 2.4 would be relevant.

Reply:
So step 3 only applies if the Subscription is not ready? Then it's basically a substep to step 2.2, because readers see that message after running the line 48 command? So we could drop the number `3. ` there and 2.4 would be the new 2.3.

Basically, line 51 would be:

> [3 blank spaces] If the status of the Subscription informs you that the sink is not a valid cluster local svc,  investigate the Subscription sink. Verify if the sink is a ...

...and line 54 would be:

> 3. If the Subscription **_is_** ready, check if the event type...

Reply:
I have updated step 2.3 (old line 53). For line 54, it is valid for both cases if the Subscription is ready or not.